cmd/eksctl/create.go

```diff
@@ -74,6 +75,8 @@ func createClusterCmd() *cobra.Command {
 	fs.IntVarP(&cfg.MinNodes, "nodes-min", "m", 0, "minimum nodes in ASG")
 	fs.IntVarP(&cfg.MaxNodes, "nodes-max", "M", 0, "maximum nodes in ASG")
 
+	fs.StringSliceVar(&availabilityZones, "zones", nil, "(auto-select if unspecified)")
```

Comment:
I like the addition of being able to explicitly set the zones. I know this is the approacvh we'd take.